### PR TITLE
Obsoleted 4 metal ion homeostasis terms. Fixes #24570

### LINKED ISSUE #24570: Obsoletion notice: GO:ID GO:0055065 metal ion and GO:0055076 transition metal ion homeostasis
Dear all, 

The proposal has been made to obsolete 
* GO:0055065 metal ion 
* GO:0055076 transition metal ion homeostasis
*  GO:0046916 cellular transition metal ion homeostasis 
*  GO:0006875 cellular metal ion homeostasis

The reason for obsoletion is that these are unnecessary grouping classes, that dont provide additional classification compared to the parent GO:0030003 intracellular cation homeostasis.

*  GO:0046916 cellular transition metal ion homeostasis  and  GO:0006875 cellular metal ion homeostasis >> can be replaced_by GO:0030003 intracellular cation homeostasis.
*  GO:0055065 metal ion  GO:0055076 transition metal ion homeostasis >> consider GO:0030003 intracellular cation homeostasis 

There are 32 EXP annotations directly to these terms, see https://github.com/geneontology/go-annotation/issues/4379 

Group | Number of annotations
-- | --
CGD | 3
FlyBase | 8
MGI | 6
PseudoCAP | 2
RGD | 2
SGD | 3
TAIR | 8

You can comment on the ticket: https://github.com/geneontology/go-ontology/issues/24570

We'd like to obsolete these terms by December 20th, 2022.

Thanks, Pascale

### PR DESCRIPTION
**Pull request by raymond91125**

### CODE REVIEW

**Comment by raymond91125** [comment]
@balhoff Need help. I cannot figure out the cause of error. Thanks.

File: src/ontology/go-edit.obo [hunk 1]
```diff
@@ -337247,14 +337249,13 @@ intersection_of: regulates_levels_of CHEBI:17996 ! chloride
...
-subset: goslim_generic
-subset: goslim_pombe
-subset: prokaryote_subset
-intersection_of: GO:0048878 ! chemical homeostasis
-intersection_of: regulates_levels_of CHEBI:25213 ! metal cation
+def: "OBSOLETE. Any process involved in the maintenance of an internal steady state of metal ions within an organism or cell." [GOC:ai, GOC:jid, GOC:mah]
+comment: This is an unnecessary grouping class.
+property_value: term_tracker_item https://github.com/geneontology/go-ontology/issues/24570 xsd:anyURI
+is_obsolete: true
+consider: 'intracellular cation homeostasis'
```
Comment (balhoff) [inline src/ontology/go-edit.obo:337258]: @raymond91125 here's the problem - term label instead of ID.

File: src/ontology/go-edit.obo [hunk 2]
```diff
@@ -337351,12 +337352,13 @@ intersection_of: regulates_levels_of CHEBI:29103 ! potassium(1+)
...
-is_a: GO:0055065 ! metal ion homeostasis
-intersection_of: GO:0048878 ! chemical homeostasis
-intersection_of: regulates_levels_of CHEBI:33515 ! transition element cation
+def: "OBSOLETE. Any process involved in the maintenance of an internal steady state of transition metal ions within an organism or cell. A transition metal is an element whose atom has an incomplete d-subshell of extranuclear electrons, or which gives rise to a cation or cations with an incomplete d-subshell. Transition metals often have more than one valency state. Biologically relevant transition metals include vanadium, manganese, iron, copper, cobalt, nickel, molybdenum and silver." [GOC:jid, GOC:mah, ISBN:0198506732]
+comment: This is an unnecessary grouping class.
+property_value: term_tracker_item https://github.com/geneontology/go-ontology/issues/24570 xsd:anyURI
+is_obsolete: true
+consider: 'intracellular cation homeostasis'
```
Comment (balhoff) [inline src/ontology/go-edit.obo:337361]: Same problem here.

**Review by balhoff — DISMISSED** [review]

File: src/ontology/go-edit.obo (hunk 1 above)
Comment (raymond91125) [reply src/ontology/go-edit.obo:337258]: ah, i see.

**Pull request merged**